modules/remotecontrol/Receiver.js

```diff
@@ -30,7 +33,11 @@ const logger = getLogger(__filename);
  *
  * @type {Transport}
  */
-const transport = getJitsiMeetTransport();
+const transport = new Transport({
+    backend: new PostMessageTransportBackend({
+        postisOptions: { scope: 'jitsi-remote-control' }
```

Comment:
So the bug is:
1) `getJitsiMeetTransport` creates transport with a dynamic scope from the URL if the iframe API is used (it will be something like jitsi_meet_api_0)
2) In the same time the remote control was working without any scope in the electron app.

1) and 2) result in breaking the communication for the remote control because different values for scope are used from both sides - Jitsi Meet and the electron app.

Now we have a new scope only for the remote control messages.